packages/component-base/test/slot-controller.test.js

```diff
@@ -0,0 +1,182 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { ControllerMixin } from '../src/controller-mixin.js';
+import { SlotController } from '../src/slot-controller.js';
+
+customElements.define(
+  'slot-controller-element',
+  class extends ControllerMixin(PolymerElement) {
+    static get template() {
+      return html`
+        <slot name="foo"></slot>
+        <slot></slot>
+      `;
+    }
+  }
+);
+
+describe('slot-controller', () => {
+  let element, child, controller;
+
+  describe('named slot', () => {
```

Comment:
Unit tests for the `slotInitializer` logic seem to be missing for both the `named` and `un-named` case.

Reply:
Good catch, will add missing tests.

Reply:
Done.

Reply:
Added.